tests/bus.rs

```diff
@@ -170,13 +170,14 @@ impl<T: Clone> BusQueues<T> {
                     if self.private[unicast.id].len() < MAX_SIZE {
                         self.private[unicast.id].push(unicast.data);
                     } else {
-                        eyre::eyre!("Cannot send data to client {}, queue is full!", unicast.id);
+                        // drop packet
                     }
                 } else {
-                    eyre::eyre!("Cannot send data to client {}", unicast.id);
+                    // drop packet
                 }
             }
         }
+        Ok(())
```

Comment:
this function just return `Ok(())` in any cases. Your caller function should handle that and return `Ok(())` instead.

Reply:
I removed it